src/main/resources/templates/signup.html

```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html lang="en">
```

Comment:
This is not part of your requirements. Radu already worked here.
Please remove it